### PR TITLE
Activating bot

### PR DESCRIPTION
**Pull request by juan-voiq**

### CODE REVIEW

File: src/components/Voicebots/EditVoicebot/Setup/Setup.jsx [hunk 1]
```diff
@@ -1,15 +1,49 @@
...
+      }
+    );
+  }
+
+  const updateGenerateProgress = () => {
+    VoicebotResponsesService.progress(
+      props.voicebotId,
+      history,
+      (response) => {
+        console.log(response);
```
Comment (DanielOlarte) [inline src/components/Voicebots/EditVoicebot/Setup/Setup.jsx:35]: @juan-voiq Are we leaving this console.log?

File: src/components/Voicebots/EditVoicebot/Setup/Train/StandardFaqs/AnswerRow.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import IconButton from "@material-ui/core/IconButton";
+import DeleteIcon from "@material-ui/icons/Delete";
+import CloudDoneIcon from '@material-ui/icons/CloudDone';
+
+const AnswerRow = (props) => {
+  const history = useHistory();
+  const [answerText, setAnswer] = useState(props.answers.current[props.answersIndex].response || "");
+
+  const onAnswerBlur = (event) => {
+    console.log(event.target);
```
Comment (DanielOlarte) [inline src/components/Voicebots/EditVoicebot/Setup/Train/StandardFaqs/AnswerRow.jsx:17]: @juan-voiq A couple of console.log

File: src/services/IntentResponsesService.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
 const axios = require('axios');
 const errorsHelper = require('../helpers/ErrorsHelper');
+const config = require('../config/voiq.json');
 
 axios.defaults.withCredentials = true;
 
 module.exports.create = (intentId, voicebotId, responseText, history, responseCallback) => {
   axios.post(
-    "http://localhost:4000/api/intent_responses",
+    config.apiUrl+"/api/intent_responses",
```
Comment (DanielOlarte) [inline src/services/IntentResponsesService.js:9]: @juan-voiq should we fix the spacing?

File: src/services/VoicebotResponsesService.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+const axios = require('axios');
+const errorsHelper = require('../helpers/ErrorsHelper');
+const config = require('../config/voiq.json');
+
+axios.defaults.withCredentials = true;
+
+module.exports.generateAll = (voicebotId, history, responseCallback) => {
+  axios.post(
+    config.apiUrl+"/api/voicebots/responses",
```
Comment (DanielOlarte) [inline src/services/VoicebotResponsesService.js:9]: @juan-voiq Same here with the spacing

File: src/config/voiq.json.sample [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "apiUrl": "http://voiq-widget-setup-be:3000"
```
Comment (DanielOlarte) [inline src/config/voiq.json.sample:2]: @juan-voiq You're missing the config value for the widget URL

File: src/components/Container.jsx [hunk 6]
```diff
@@ -13,6 +13,7 @@ import EditVoicebot from "./Voicebots/EditVoicebot/EditVoicebot";
 const PrivateRoute = ({children, ...props}) => {
   const history = useHistory();
 
+  // TODO: Verify what happens if role doesn't have access
```
[the remark this answers is not in the document]
Comment (juan-voiq) [reply src/components/Container.jsx:16]: Want to go over this in testing

**Pull request merged**